### PR TITLE
Add the properties bitrate_mode, encoder_info and encoder_settings

### PR DESCRIPTION
**Pull request by jcassette**

### CODE REVIEW

File: mediafile.py [hunk 1]
```diff
@@ -2326,6 +2327,20 @@ def bitrate(self):
                 return 0
             return int(self.filesize * 8 / self.length)
 
+    @property
+    def bitrate_mode(self):
+        """The mode of the bitrate used in the audio coding (a string).
```
Comment (sampsyo) [inline mediafile.py:2332]: Perhaps we should mention what the current possibilities are? (Without promising that they are the only possibilities this field will ever contain.)

File: mediafile.py [hunk 2]
```diff
@@ -2326,6 +2327,20 @@ def bitrate(self):
...
+        Only available for certain file formats (empty where unavailable).
```
Comment (sampsyo) [inline mediafile.py:2333]: We could "spoil the surprise" and admit that the only format it currently works for is MP3.

**Review by sampsyo — COMMENTED** [review]
Seems great overall; thanks for the addition! Do you think this needs to be added to the docs anywhere? Or perhaps the new docstring is enough?

File: mediafile.py (hunk 1 above)
Comment (jcassette) [reply mediafile.py]: Sure, I have updated the docstring

File: mediafile.py (hunk 2 above)
Comment (jcassette) [reply mediafile.py]: Sure, I have updated the docstring

**Comment by jcassette** [comment]
Thanks, I have updated the doc index :-)

**Comment by jcassette** [comment]
I guess I can also add the fields `encoder_info` and `encoder_settings` since they are available.
Would you like this included in this PR?

**Comment by jcassette** [comment]
I have also added the properties encoder_info and encoder_settings since they are available.
This will allow to get information about the VBR quality, even though it would be better if mutagen exposed it directly.

**Comment by sampsyo** [comment]
Nice; this all seems great!!

The style checker has one suggestion: a few of the places where there's a dictionary literal, like this:

```py
'encoder_info' : '',
```

Should have one less space, like this:

```py
'encoder_info': '',
```

Otherwise, I think this is good to go. Thanks again for sorting this out!

**Comment by jcassette** [comment]
Oops, I removed the extra spaces, thank you!

**Comment by sampsyo** [comment]
Awesome; thanks!! I bumped this to v0.9.0 to indicate a new feature, but I think it's good to go.

**Pull request merged**